pytorch_lightning/callbacks/device_stats_monitor.py

```diff
@@ -32,6 +33,9 @@ class DeviceStatsMonitor(Callback):
     Automatically monitors and logs device stats during training stage. ``DeviceStatsMonitor``
     is a special callback as it requires a ``logger`` to passed as argument to the ``Trainer``.
 
+    Args:
+        cpu_stats: if True, will also record CPU stats in addition to accelerator stats (default: True).
```

Comment:
```suggestion
        cpu_stats: if True, will also record CPU stats in addition to accelerator stats.
```

Reply:
I moved to match the formatting in `trainer.py`. Is that okay?